johnwayodi/sm-api
# #161406423 Split models to individual files

#### What does this PR do?
Splits models to their individual files
#### Description of Task to be completed?
Ensure tests are still running successfully after restructure
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api.git`
2. Ensure you are in the cloned repo, **_sm-api_** by default.
3. `git checkout ch-models-restructure-161406423`
4. Run the tests.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161406423
#### Questions:
Any feedback on what to improve?
